arch/dev/cc1200/cc1200-conf.h

```diff
@@ -72,6 +72,8 @@
 #else
 #define CC1200_MAX_PAYLOAD_LEN           127
 #endif
+
+#define cc1200_driver_max_payload_len    CC1200_MAX_PAYLOAD_LEN
```

Comment:
Shouldn't this be 125? Won't this result in 127+2 bytes CRC frames, even without 802.15.4g?

Reply:
Not sure - you're the expert on this radio! I got the impression  that 127 is ok after reading the comment above